recipe/meta.yaml

```
@@ -1,25 +1,31 @@
-{% set name = "cni" %}
-{% set version = "0.6.0" %}
+{% set import_path = "github.com/containernetworking/cni" %}
+{% set version = "0.7.1" %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/' + import_path).replace("/", "\\" if win else "/") %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/containernetworking/{{ name }}/archive/v{{ version }}.tar.gz
-  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
-  sha256: 802f4a002b4eb774624a9dc1c859d3c9926eb2d862e66a673fc99cfc8bcd7494
+  folder: {{ pkg_src }}
+  url: https://{{ import_path }}/archive/v{{ version }}.tar.gz
+  sha256: 4517eabfd65aea2012dc48d057bf889a0a41ed9837387d95cd1e36c0dbddcfd4
 
 build:
   skip: True  # [not linux]
-  number: 1002
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v  ./cnitool
 
 requirements:
   build:
     - {{ compiler('go') }}
     - {{ compiler('c') }}
-    - make
   host:
+    - go
```

Go is already selected as a compiler in L25, so it is not needed here.
```suggestion
    - go
```